x-pack/legacy/plugins/reporting/export_types/common/lib/screenshots.ts

```diff
@@ -83,14 +117,20 @@ export function screenshotsObservableFactory(server) {
     );
   };
 
-  const getNumberOfItems = async (browser, layout) => {
+  const getNumberOfItems = async (
+    browser: HeadlessBrowser,
+    layout: LayoutInstance
+  ): Promise<number> => {
     // returns the value of the `itemsCountAttribute` if it's there, otherwise
     // we just count the number of `itemSelector`
     const itemsCount = await browser.evaluate({
       fn: (selector, countAttribute) => {
         const elementWithCount = document.querySelector(`[${countAttribute}]`);
-        if (elementWithCount) {
-          return parseInt(elementWithCount.getAttribute(countAttribute));
+        if (elementWithCount && elementWithCount != null) {
```

Comment:
isn't this redundant to check for null?

Reply:
this one is, yes: since document.querySelector either returns an element (always truthy) or null

Reply:
https://github.com/elastic/kibana/pull/42102/commits/bd164cf0f68364f547eb4d8b7226b9d2a02eef42